docker-compose.yml

```diff
@@ -4,10 +4,10 @@ services:
   cht-user-management:
     image: public.ecr.aws/medic/cht-user-management:latest
     environment:
-      - EXTERNAL_PORT=${EXTERNAL_PORT:-3000}
+      - EXTERNAL_PORT=${EXTERNAL_PORT:-3500}
       - COOKIE_PRIVATE_KEY=${COOKIE_PRIVATE_KEY}
       - CONFIG_NAME=${CONFIG_NAME}
     ports:
-      - '${EXTERNAL_PORT-3000}:${PORT:-3000}'
+      - '${EXTERNAL_PORT-3500}:${PORT:-3000}'
```

Comment:
ugh - this is my fault - I told you to make these changes, but we should have just made then in the in the `.env` and let this file stay as is.  We're changing the default values which we don't need to do.

Reply:
I note these are still changed to `3500`.  please revert the both to `3000`.